dev/buildtool/support_functions.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
```

Comment:
Any particular reason for orchestrating this in bash over python?

Reply:
It's growing organically. Maybe eventually python. It was pretty simplistic except for the process array. It might tip the complexity scale when I write the publishing script. But for now the flow_*.sh looks a lot easier to read and understand than I imagine python would. 

I can switch now if you think I should.

Reply:
Just pointing it out so you're aware. I think this time around things are logically separated and will do enough reporting so that we may not have the issues we've had with orchestration in bash in the past.

Fine for now, we can port it if we need to.